### PR TITLE
Remove explict allowance checks in treasury

### PR DESCRIPTION
**Pull request by doerfli**

### CODE REVIEW

File: tests/test_treasury_module.py [hunk 1]
```diff
@@ -94,6 +105,179 @@ def test_bundle_creation_with_riskpool_wallet_not_set(
...
+        oracleProvider,
+        riskpoolKeeper,
+        capitalOwner,
+        True
+    )
+
+    # prepare product and riskpool
+    safetyFactor = 2
+    amount = 10000
+    testCoin.transfer(capitalOwner, safetyFactor * amount, {'from': owner})
```
Comment (matthiaszimmermann) [inline tests/test_treasury_module.py:227]: please check if capitalOwner represents the riskpool wallet, if this is the case change test case to riskpoolKeeper as investor

File: tests/test_treasury_module.py [hunk 2]
```diff
@@ -94,6 +105,179 @@ def test_bundle_creation_with_riskpool_wallet_not_set(
...
+    product.confirmClaim(policyId, claimId, claimAmount, {'from': productOwner})
+
+    claim = instanceService.getClaim(policyId, claimId).dict()
+    assert claim["state"] ==  1 # enum ClaimState {Applied, Confirmed, Declined, Closed}
+
+    # check that it's possible to create payout for claim in confirmed state
+    payoutAmount = claimAmount
+    
+    # prepare allowance for payout that is too small
+    testCoin.approve(instance.getTreasury(), payoutAmount * 0.9, {'from': productOwner})
```
Comment (matthiaszimmermann) [inline tests/test_treasury_module.py:274]: the approval needs to come from the riskpoolWallet, not the productOwner

File: tests/test_treasury_module.py [hunk 3]
```diff
@@ -94,6 +105,179 @@ def test_bundle_creation_with_riskpool_wallet_not_set(
...
+        oracleProvider,
+        riskpoolKeeper,
+        capitalOwner,
+        True
+    )
+
+    # prepare allowance too small for riskpool funding 
+    safetyFactor = 2
+    amount = 10000
+    testCoin.transfer(capitalOwner, safetyFactor * amount, {'from': owner})
```
Comment (matthiaszimmermann) [inline tests/test_treasury_module.py:134]: verify that capitalOwner is not the riskpoolWallet. please use the riskpoolKeeper as the investor for the test case, not the capitalOwner

File: tests/test_treasury_module.py [hunk 4]
```diff
@@ -94,6 +105,179 @@ def test_bundle_creation_with_riskpool_wallet_not_set(
...
+        oracleProvider,
+        riskpoolKeeper,
+        capitalOwner,
+        True
+    )
+
+    # fund bundle
+    safetyFactor = 2
+    amount = 10000
+    testCoin.transfer(capitalOwner, safetyFactor * amount, {'from': owner})
```
Comment (matthiaszimmermann) [inline tests/test_treasury_module.py:171]: same as above

**Review by matthiaszimmermann — CHANGES_REQUESTED** [review]

File: tests/test_treasury_module.py (hunk 3 above)
Comment (doerfli) [reply tests/test_treasury_module.py]: done

File: tests/test_treasury_module.py (hunk 2 above)
Comment (doerfli) [reply tests/test_treasury_module.py]: done

File: tests/test_treasury_module.py (hunk 1 above)
Comment (doerfli) [reply tests/test_treasury_module.py]: done

**Review by matthiaszimmermann — APPROVED** [review]

**Pull request merged**